installscripts/Installer.sh

```diff
@@ -61,99 +41,230 @@ spin_wheel()
+
+  #Get Jazz Installer code base
+  sudo rm -rf jazz-installer
+  git clone -b $JAZZ_BRANCH https://github.com/tmobile/jazz-installer.git >>$LOG_FILE 2>&1 &
+  spin_wheel $! "Downloading jazz Installer"
+
+  #Download and Install wget
+  sudo yum install -y wget >>$LOG_FILE 2>&1 &
+  spin_wheel $! "Downloading and install wget"
+
+  #Download and install pip
+  sudo wget -q https://bootstrap.pypa.io/get-pip.py
+  sudo python get-pip.py >>$LOG_FILE 2>&1 &
+  spin_wheel $! "Downloading and install pip"
+
+  #Download and install paramiko
+  sudo pip install paramiko >>$LOG_FILE 2>&1 &
+  spin_wheel $! "Downloading and install paramiko"
+}
+
+function install_packages_verbose () {
+  # Download and Installing Softwares required for Jazz Installer
+  # 1. GIT
+  # 2. Java Jdk - 8u112-linux-x64
+  # 3. Unzip
+  # 4. AWSCLI
+  # 5. Terraform - 0.9.11
+  # 6. JQ - 1.5
+  # 7. Atlassian CLI - 6.7.1
+
+  # Install git
+  sudo yum install -y git | tee -a $LOG_FILE
+
+  # Download and Install java
+  curl -v -j -k -L -H "Cookie: oraclelicense=accept-securebackup-cookie" http://download.oracle.com/otn-pub/java/jdk/8u131-b11/d54c1d3a095b4ff2b6607d096fa80163/jdk-8u131-linux-x64.rpm -o jdk-8u112-linux-x64.rpm | tee -a $LOG_FILE
+  sudo rpm -ivh --force ./jdk-8u112-linux-x64.rpm | tee -a $LOG_FILE
+  sudo rm -rf jdk-8u112-linux-x64.rpm
+
+  # Download and Install unzip
+  sudo yum install -y unzip | tee -a $LOG_FILE
+
+  # Create a temporary folder .
+  # Here we will have all the temporary files needed and delete it at the end
+  sudo rm -rf $INSTALL_DIR/jazz_tmp
+  mkdir $INSTALL_DIR/jazz_tmp
+
+  # Download and Install awscli
+  sudo curl -L https://s3.amazonaws.com/aws-cli/awscli-bundle.zip -o $INSTALL_DIR/jazz_tmp/awscli-bundle.zip | tee -a $LOG_FILEs
+  sudo rm -rf $INSTALL_DIR/jazz_tmp/awscli-bundle
+  sudo unzip $INSTALL_DIR/jazz_tmp/awscli-bundle.zip -d $INSTALL_DIR/jazz_tmp | tee -a $LOG_FILE
+  sudo rm -rf /usr/local/aws
+  sudo rm -f /usr/local/bin/aws
+
+  cd $INSTALL_DIR/jazz_tmp/awscli-bundle/
+  sudo ./install -i /usr/local/aws -b /usr/local/bin/aws | tee -a $LOG_FILE
+  cd $INSTALL_DIR/
+
+  #Download and Install Terraform
+  sudo curl -v -L https://releases.hashicorp.com/terraform/0.9.11/terraform_0.9.11_linux_amd64.zip?_ga=2.191030627.850923432.1499789921-755991382.1496973261 -o $INSTALL_DIR/jazz_tmp/terraform.zip | tee -a $LOG_FILE
+  sudo unzip -o $INSTALL_DIR/jazz_tmp/terraform.zip -d /usr/bin | tee -a $LOG_FILE
+
+  #Downloading and Install atlassian-cli
+  sudo curl -L https://bobswift.atlassian.net/wiki/download/attachments/16285777/atlassian-cli-6.7.1-distribution.zip -o $INSTALL_DIR/jazz_tmp/atlassian-cli-6.7.1-distribution.zip | tee -a $LOG_FILE
+  sudo unzip -o $INSTALL_DIR/jazz_tmp/atlassian-cli-6.7.1-distribution.zip | tee -a $LOG_FILE
+
+  #Get Jazz Installer code base
+  sudo rm -rf jazz-installer
+  git clone -b $JAZZ_BRANCH https://github.com/tmobile/jazz-installer.git | tee -a $LOG_FILE
```

Comment:
Can we extract the Git URL to a top-level variable as well?

Reply:
This is done! :+1: